Dockerfile

```diff
@@ -0,0 +1,5 @@
+FROM python:3.8-slim-buster
+WORKDIR /chatops
+COPY . .
+RUN chmod +x script.sh
+CMD cd /chatops && ./script.sh
```

Comment:
@NaveenKundeti-IBS  Please `cd` once either in the `Dockerfile` or the `script`

Reply:
yes updated in the script file sir @tomegathericon